doc/library/xtensor/module_functions.md

```diff
@@ -0,0 +1,11 @@
+(libdoc_xtensor_module_function)=
+# `xtensor` -- Module level operations
+
+```{eval-rst}
+.. automodule:: pytensor.xtensor
+.. autosummary::
+   :toctree: generated/
+
+   concat
+   dot
```

Comment:
given this will generate specific pages for `concat` and `dot` which diverges from the behaviour of the other api pages for the sibling modules, it might look better to do

```suggestion
.. automodule:: pytensor.xtensor
   :members: concat, dot
```